platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/ShadowgraphByBirthRoundTests.java

```diff
@@ -143,43 +142,43 @@ private void initShadowGraph(final Random random, final int numEvents, final int
     @ParameterizedTest
     @MethodSource("graphSizes")
     void testFindAncestorsForMultipleEvents(final int numEvents, final int numNodes) {
```

Comment:
Oh!  is this whole file a clone of `ShadowGraphTest.java` but using birthRound for ancient?

Reply:
Yup!

This test already had a lot of parameterized tests, and it was going to be really complex to add an additional dimension to the parameter space.